gnovm/pkg/gnolang/store.go

```diff
@@ -72,6 +77,7 @@ type defaultStore struct {
 	baseStore        store.Store           // for objects, types, nodes
 	iavlStore        store.Store           // for escaped object hashes
 	pkgInjector      PackageInjector       // for injecting natives
```

Comment:
So... not yet.

Even if we don't use it in vm/builtins.go, we use it in tests/imports.go. As you can see, it's basically used in one single case: strconv.ParseInt and strconv.NumError.

I'm going to address this in a different PR. Adding `strconv` is high on my stdlib tasklist anyway. This should avoid creating headaches with trying to link up the type.

I do think that since I actually added "type-linking", we could remove it here. But this PR is already gigantic as it is. So expect me to do it for the upcoming strconv pr.

Reply:
(as far as nuances go, the big one is that nativeStore allows fns to be called within the same package as they are declared, while pkgInjector does not; this was the reason for horrors such as internal/math).